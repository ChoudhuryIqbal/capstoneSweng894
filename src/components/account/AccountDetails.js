import React from 'react'
import { TextInput } from 'react-native'

const AccountDetails = (props) => {
    return(

    )
}

export default AccountDetails