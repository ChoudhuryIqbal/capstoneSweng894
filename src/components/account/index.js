export * from './AccountButtons'
export * from './AccountDetails'
export * from './AccountAddress'
export * from './AccountForm'
export * from './AccountImage'
export * from './AccountImageUpload'
export * from './AccountLogIn'
export * from './AccountOptions'