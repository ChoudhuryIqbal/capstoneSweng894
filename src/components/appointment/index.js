export * from './AppointmentList'
export * from './AppointmentItem'
export * from './AppointmentDetailView'
export * from './AppointmentEditView'
export * from './AppointmentView'
export * from './AppointmentRenderPickerField'
export * from './BusinessName'
export * from './Calendar'
export * from './Time'
export * from './Status'
export * from './Stylist'
export * from './Service'