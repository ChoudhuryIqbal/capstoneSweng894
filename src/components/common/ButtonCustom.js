import React from 'react';
import { Text, TouchableOpacity } from 'react-native';

const ButtonCustom = ({ onPress, children, buttonStyle, textStyle }) => {
return (
    <TouchableOpacity onPress={onPress} style={buttonStyle}>
      <Text style={textStyle}>
        {children}
      </Text>
    </TouchableOpacity>
  );
};

export {ButtonCustom}
