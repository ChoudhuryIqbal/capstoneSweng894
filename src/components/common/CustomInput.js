import React from 'react';
import { Platform, TextInput, View, Text } from 'react-native';

const CustomInput = ({ value, onChangeText, placeholder, 
     inputStyle, containerStyle, textAlign}) => {
  return (
    <View style={containerStyle} >
      <TextInput
        placeholder={placeholder}
        underlineColorAnroid={Platform.Os === 'ios' ? '' : 'transparent'}
        autoCorrect={false}
        style={inputStyle}
        value={value}
        textAlign={textAlign}
        onChangeText={onChangeText}
      />
    </View>
  );
};


export { CustomInput };
