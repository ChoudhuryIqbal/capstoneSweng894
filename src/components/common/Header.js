import React from 'react';
import {Platform,View,Text} from 'react-native';
 

const Header = ({}) => {
return (
<View style={styles.headerRow}>
<View style={{alignItems:'flex-start'}}>
    <Text style={styles.headerText}>{"Enter Your Location"}</Text>
</View>
</View>

);
};
export default {Header}