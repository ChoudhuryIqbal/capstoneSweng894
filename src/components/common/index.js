export * from './Button';
export * from './Card';
export * from './CardSection';
export * from './Header';
export * from './Input';
export * from './InputCustom';
export * from './Spinner';
export * from './InputNumber';
export * from './ImageButton';
export * from './ButtonCustom';
export * from './CheckBox';
export * from './PickerList';
export * from './InputNumberCustom'
export * from './Time'