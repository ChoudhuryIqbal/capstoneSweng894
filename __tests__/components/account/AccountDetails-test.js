import React from 'react'
import AccountDetails from '../../../src/components/account/AccountDetails'
import {ButtonCustom} from '../../../src/components/common/'
import {create} from 'react-test-renderer';


describe("AccountDetail correctly", () => {
    test("Matches the snapshot", () => {
        const onRenderProfileButton = () => {
            return (
                <ButtonCustom>
                    {'Submit'}
                </ButtonCustom>
            )
        }

        const onRenderProfileDelete = () => {
            return (
                <ButtonCustom>
                    {'Delete Profile'}
                </ButtonCustom>
            )
        }

        const onRenderPreference = () => {
            return (
                <ButtonCustom>
                    {'Preference'}
                </ButtonCustom>
            )
        }

        const detail = create(
            <AccountDetails
                Creation={false}
                Deletion={"Test"}
                Preference={"Test"}
                firstName={"Test"}
                onFirstNameChge={() => {}}
                errorFirstName={"Test"}
                lastName={"Test"}
                onLastNameChge={() => {}}
                errorLastName={"Test"}
                email={"Test"}
                onEmailChge={() => {}}
                errorEmail={"Test"}
                password={"Test"}
                onPasswordChge={() => {}}
                errorPassword={"Test"}
                telephone={"Test"}
                onTelephoneChge={() => {}}
                errorTelephone={"Test"}
                dob={"Test"}
                ondobChge={() => {}}
                errorDob={"Test"}
                gender={"Test"}
                onGenderChge={() => {}}
                errorGender={"Test"}
                address={"Test"}
                onAddressChge={() => {}}
                errorAddress={"Test"}
                city={"Test"}
                onCityChge={() => {}}
                errorCity={"Test"}
                state={"Test"}
                onStateChge={() => {}}
                errorState={"Test"}
                zip={"Test"}
                onZipChge={() => {}}
                errorZip={"Test"}
                isProvider={"Test"}
                onProviderChge={() => {}}
                errorIsProvider={false}
                error={"Test"}
                onSubmit={() => onRenderProfileButton()}
                onDelete={() => onRenderProfileDelete()}
                onPref={() => onRenderPreference()}
            />
        );
        expect (detail.toJSON()).toMatchSnapshot();
    })
  });