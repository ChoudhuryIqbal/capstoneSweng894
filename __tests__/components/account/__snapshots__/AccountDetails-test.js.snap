// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountDetail correctly Matches the snapshot 1`] = `
<View
  style={
    Object {
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "borderColor": "black",
        "borderRadius": 10,
        "borderWidth": 1,
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        First Name: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. John"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Last Name: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. Williams"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "borderColor": "black",
        "borderRadius": 10,
        "borderWidth": 1,
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Telephone: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. 111-111-1111"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Date of Birth: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.ei 1987-01-01"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "borderColor": "black",
        "borderRadius": 10,
        "borderWidth": 1,
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "margin": 5,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Gender: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. Male"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      >
        Test
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "backgroundColor": "#ffffff",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "flex": 1,
            "fontSize": 16,
            "paddingBottom": 10,
          }
        }
      >
        Is Provider: 
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        onChangeText={[Function]}
        placeholder="i.e. Yes/No"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "color": "#000",
            "elevation": 1,
            "flex": 2,
            "fontSize": 18,
            "lineHeight": 3,
            "paddingBottom": 20,
            "paddingLeft": 5,
            "paddingRight": 5,
            "shadowOffset": Object {
              "height": 10,
              "width": 5,
            },
            "shadowOpacity": 1,
            "shadowRadius": 5,
          }
        }
        underlineColorAndroid="transparent"
        underlineColorAnroid="transparent"
        value="Test"
      />
      <Text
        style={
          Object {
            "bottom": 0,
            "color": "red",
            "elevation": 1,
            "fontSize": 10,
            "justifyContent": "center",
            "marginTop": 10,
            "position": "absolute",
            "textAlign": "center",
          }
        }
      />
    </View>
  </View>
  <View>
    <View
      style={
        Object {
          "borderColor": "black",
          "borderRadius": 10,
          "borderWidth": 1,
          "display": "flex",
          "flex": 1,
          "flexDirection": "row",
          "margin": 5,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "backgroundColor": "#ffffff",
            "display": "flex",
            "flex": 1,
            "flexBasis": 90,
            "flexDirection": "column",
            "justifyContent": "center",
            "margin": 5,
          }
        }
      >
        <Text
          style={
            Object {
              "flex": 1,
              "fontSize": 16,
              "paddingBottom": 10,
            }
          }
        >
          Address: 
        </Text>
        <TextInput
          allowFontScaling={true}
          autoCorrect={false}
          onChangeText={[Function]}
          placeholder="i.e. 100 Main Street"
          rejectResponderTermination={true}
          style={
            Object {
              "backgroundColor": "#ffffff",
              "color": "#000",
              "elevation": 1,
              "flex": 2,
              "fontSize": 18,
              "lineHeight": 3,
              "paddingBottom": 20,
              "paddingLeft": 5,
              "paddingRight": 5,
              "shadowOffset": Object {
                "height": 10,
                "width": 5,
              },
              "shadowOpacity": 1,
              "shadowRadius": 5,
            }
          }
          underlineColorAndroid="transparent"
          underlineColorAnroid="transparent"
          value="Test"
        />
        <Text
          style={
            Object {
              "bottom": 0,
              "color": "red",
              "elevation": 1,
              "fontSize": 10,
              "justifyContent": "center",
              "marginTop": 10,
              "position": "absolute",
              "textAlign": "center",
            }
          }
        >
          Test
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "backgroundColor": "#ffffff",
            "display": "flex",
            "flex": 1,
            "flexBasis": 90,
            "flexDirection": "column",
            "justifyContent": "center",
            "margin": 5,
          }
        }
      >
        <Text
          style={
            Object {
              "flex": 1,
              "fontSize": 16,
              "paddingBottom": 10,
            }
          }
        >
          City: 
        </Text>
        <TextInput
          allowFontScaling={true}
          autoCorrect={false}
          onChangeText={[Function]}
          placeholder="i.e. Malven"
          rejectResponderTermination={true}
          style={
            Object {
              "backgroundColor": "#ffffff",
              "color": "#000",
              "elevation": 1,
              "flex": 2,
              "fontSize": 18,
              "lineHeight": 3,
              "paddingBottom": 20,
              "paddingLeft": 5,
              "paddingRight": 5,
              "shadowOffset": Object {
                "height": 10,
                "width": 5,
              },
              "shadowOpacity": 1,
              "shadowRadius": 5,
            }
          }
          underlineColorAndroid="transparent"
          underlineColorAnroid="transparent"
          value="Test"
        />
        <Text
          style={
            Object {
              "bottom": 0,
              "color": "red",
              "elevation": 1,
              "fontSize": 10,
              "justifyContent": "center",
              "marginTop": 10,
              "position": "absolute",
              "textAlign": "center",
            }
          }
        >
          Test
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "borderColor": "black",
          "borderRadius": 10,
          "borderWidth": 1,
          "display": "flex",
          "flex": 1,
          "flexDirection": "row",
          "margin": 5,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "backgroundColor": "#ffffff",
            "display": "flex",
            "flex": 1,
            "flexBasis": 90,
            "flexDirection": "column",
            "justifyContent": "center",
            "margin": 5,
          }
        }
      >
        <Text
          style={
            Object {
              "flex": 1,
              "fontSize": 16,
              "paddingBottom": 10,
            }
          }
        >
          State: 
        </Text>
        <TextInput
          allowFontScaling={true}
          autoCorrect={false}
          onChangeText={[Function]}
          placeholder="i.e. PA"
          rejectResponderTermination={true}
          style={
            Object {
              "backgroundColor": "#ffffff",
              "color": "#000",
              "elevation": 1,
              "flex": 2,
              "fontSize": 18,
              "lineHeight": 3,
              "paddingBottom": 20,
              "paddingLeft": 5,
              "paddingRight": 5,
              "shadowOffset": Object {
                "height": 10,
                "width": 5,
              },
              "shadowOpacity": 1,
              "shadowRadius": 5,
            }
          }
          underlineColorAndroid="transparent"
          underlineColorAnroid="transparent"
          value="Test"
        />
        <Text
          style={
            Object {
              "bottom": 0,
              "color": "red",
              "elevation": 1,
              "fontSize": 10,
              "justifyContent": "center",
              "marginTop": 10,
              "position": "absolute",
              "textAlign": "center",
            }
          }
        >
          Test
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "backgroundColor": "#ffffff",
            "display": "flex",
            "flex": 1,
            "flexBasis": 90,
            "flexDirection": "column",
            "justifyContent": "center",
            "margin": 5,
          }
        }
      >
        <Text
          style={
            Object {
              "flex": 1,
              "fontSize": 16,
              "paddingBottom": 10,
            }
          }
        >
          Zip: 
        </Text>
        <TextInput
          allowFontScaling={true}
          autoCorrect={false}
          keyboardType="numbers-and-punctuation"
          onChangeText={[Function]}
          placeholder="i.e. 19901"
          rejectResponderTermination={true}
          style={
            Object {
              "backgroundColor": "#ffffff",
              "color": "#000",
              "elevation": 1,
              "flex": 2,
              "fontSize": 18,
              "lineHeight": 3,
              "paddingBottom": 20,
              "paddingLeft": 5,
              "paddingRight": 5,
              "shadowOffset": Object {
                "height": 10,
                "width": 5,
              },
              "shadowOpacity": 1,
              "shadowRadius": 5,
            }
          }
          underlineColorAndroid="transparent"
          underlineColorAnroid="transparent"
          value="Test"
        />
        <Text
          style={
            Object {
              "bottom": 0,
              "color": "red",
              "elevation": 1,
              "fontSize": 10,
              "justifyContent": "center",
              "marginTop": 10,
              "position": "absolute",
              "textAlign": "center",
            }
          }
        >
          Test
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "backgroundColor": "#ffffff",
          "color": "red",
          "fontSize": 20,
          "justifyContent": "center",
        }
      }
    >
      Test
    </Text>
  </View>
  <View
    style={
      Object {
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "paddingBottom": 15,
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text>
        Submit
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "paddingBottom": 15,
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text>
        Delete Profile
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "paddingBottom": 15,
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text>
        Preference
      </Text>
    </View>
  </View>
</View>
`;
